packages/wonder-blocks-dropdown/components/dropdown-core-virtualized.js

```diff
@@ -194,13 +226,14 @@ class DropdownCoreVirtualized extends React.Component<Props, State> {
 
     render() {
         if (!this.state.width) {
-            // first load, render non virtualized items to calculate width
+            // if we don't pass a fixed value, then we need to render
+            // non-virtualized items to calculate width
             return this.renderInitialItems();
         } else {
-            // then render the virtualized list
+            // width has been provided, then render the virtualized list
             return this.renderVirtualizedList();
         }
     }
 }
 
-export default DropdownCoreVirtualized;
+export default withActionScheduler/*:: <OwnProps> */(DropdownCoreVirtualized);
```

Comment:
Hmm - this is a bit unfortunate. I think we may need to look in to updating `withActionScheduler` with the better Flow types that @kevinbarabash figured out.

I noticed that you were able to use the correct (non-comment) Flow types for `withActionScheduler` below - why didn't it work here?

Reply:
🤦‍♂ you're right, I implemented this one before and I think I just got used to add those comments without testing this way first.